flow/core/kernel/scenario/__init__.py

```diff
@@ -1,3 +1,4 @@
 from flow.core.kernel.scenario.base import KernelScenario
+from flow.core.kernel.scenario.traci import TraCIScenario
```

Comment:
The import path here is different than the import above?

Reply:
this is the __init__ file, we're just adding the stuff to be able to call them more easily later